### PR TITLE
#164798194 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by ezrogha**
#### What does this PR do?
Integrates TravisCI with a readme badge to the project

#### Description of Task to be completed?
Integrating TravisCI to author's Heaven to enable continuous integration

#### What are the relevant pivotal tracker stories?
[#164798194](https://www.pivotaltracker.com/story/show/164798194)

### CODE REVIEW

File: .env_config_sample [hunk 1]
```diff
@@ -1,2 +1,2 @@
-export DATABASE_URI='postgres://USER:PASSWORD@HOST:PORT/NAME'
-export SECRET_KEY='AuthorsHaven'
+export DATABASE_URI='postgres://postgres:password@localhost:5432/authors_haven'
```
Comment (ja-odur) [inline .env_config_sample]: @ezrogha please revert the these changes

File: Pipfile.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "1de757cc8f7c36265d20bb78502017609e875b6028061c7daa4ce023e1a3239c"
+            "sha256": "ba714e565f1d4f9e768e63c770cb92522cd7c7a44d94c19753a3de48d0176ee7"
```
Comment (ja-odur) [inline Pipfile.lock:4]: @ezrogha do not track this file

File: authors/settings.py [hunk 3]
```diff
@@ -82,9 +82,14 @@
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
 DATABASES = {
-    'default': dj_database_url.config(
-        default=config('DATABASE_URI', '')
-    )
+    'default': {
```
Comment (ja-odur) [inline authors/settings.py]: @ezrogha please revert this. we are using the dj_database_url

**Review by ja-odur — CHANGES_REQUESTED** [review]

**Comment by dannylwe** [comment]
@ezrogha, kindly add a label indicating the state/progress of this PR

File: .env_config_sample [hunk 4]
```diff
@@ -1,2 +1,2 @@
...
+export SECRET_KEY='AuthorsHaven
```
Comment (dannylwe) [inline .env_config_sample]: @ezrogha kindly replace the quotation mark and leave a blank line at the bottom.

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: .env_config_sample (hunk 4 above)
Comment (ezrogha) [reply .env_config_sample]: Thanks @dannylwe

File: .env_config_sample (hunk 1 above)
Comment (ezrogha) [reply .env_config_sample]: Thanks for the feedback and assistance @ja-odur

**Review by felixkiryowa — APPROVED** [review]
Thanks, @ezrogha  for emitting the pipfile.lock file

**Review by ja-odur — APPROVED** [review]

**Review by dannylwe — APPROVED** [review]

**Pull request merged**